### PR TITLE
Update create-interactive-grant.mdx

### PR DESCRIPTION
**Pull request by melissahenderson**
## Changes proposed in this pull request

Initial revisions to the "Create Interactive Grant" guide for review and feedback.

## Context

After reviewing the first draft of the guides, we decided that they need to:
* Display the code snippets (as applicable) instead of API bodies
* Use a tabbed display for languages, like the Code Snippets pages
* Contain explanations for any applicable API properties so that our audience won't need to toggle between the API docs and the guides
* Mention, in some fashion, information about signature headers

### CODE REVIEW

File: docs/src/content/docs/guides/create-interactive-grant.mdx [hunk 1]
```diff
@@ -31,56 +32,62 @@ Although grants for creating incoming payments and quotes are non-interactive by
...
-```ts
-interact: {
-    start: ["redirect"],
-},
-```
+Indicate your client is capable of directing the end-user to a URI to start an interaction. We’ll use an outgoing payment grant as an example.
 
-### 2. Specify a redirect finish mode
+1. Add the `interact` object to your grant request.
+2. Specify `redirect` as the `start` mode. Open Payments only supports `redirect`. The redirect URI will be provided by the authorization server in its response.
```
Comment (sabineschaller) [inline docs/src/content/docs/guides/create-interactive-grant.mdx:63]:
```suggestion
2. Specify `redirect` as the `start` mode. Open Payments only supports `redirect` today. The redirect URI will be provided by the authorization server in its response.
```
to indicate that more is coming?

File: docs/src/content/docs/guides/create-interactive-grant.mdx [hunk 2]
```diff
@@ -31,56 +32,62 @@ Although grants for creating incoming payments and quotes are non-interactive by
...
 
-</CodeBlock>
+#### Start interaction with the end user
 
-### 3. Interact with the end user
-
-#### Start interaction
-
-Once your client receives the [authorization server’s response](/apis/auth-server/operations/post-request/#responses), it must send the end-user to the `interact redirect` URI contained in the response. This initiates the interaction flow.
+Once your client receives the authorization server’s response, it must send the end-user to the `interact redirect` URI contained in the response. This starts the interaction flow.
```
Comment (sabineschaller) [inline docs/src/content/docs/guides/create-interactive-grant.mdx:84]: We should add an example response body, I think.

```suggestion
Once your client receives the authorization server’s response, it must send the end-user to the `interact.redirect` URI contained in the response. This starts the interaction flow.
```

File: docs/src/content/docs/guides/create-interactive-grant.mdx [hunk 3]
```diff
@@ -89,22 +96,20 @@ Provided the end-user approves the grant, the authorization server:
 - Sends the end-user to your client’s previously defined `finish uri`. The means by which the server sends the end-user to the URI is out of scope, but common options include redirecting the end-user from a web page and launching the system browser with the target URI.
```
Comment (sabineschaller) [inline docs/src/content/docs/guides/create-interactive-grant.mdx:96]:
```suggestion
- Sends the end-user to your client’s previously defined `finish.uri`. The means by which the server sends the end-user to the URI is out of scope, but common options include redirecting the end-user from a web page and launching the system browser with the target URI.
```

File: docs/src/content/docs/guides/create-interactive-grant.mdx [hunk 4]
```diff
@@ -89,22 +96,20 @@ Provided the end-user approves the grant, the authorization server:
...
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-continuation.ts'
+  chunk='3'
+/>
 
 Issue the request to the `continue uri` supplied in the authorization server’s initial grant response. For example:
 
 `POST https://auth.rafiki.money/continue/49fb4a97-f880-449c-b0e9-d3ac1c947b10`
 
-A successful [continuation response](/apis/auth-server/operations/post-continue/#responses) from the authorization server provides your client with an access token and other information necessary to continue the transaction.
+A successful continuation response from the authorization server provides your client with an access token and other information necessary to continue the transaction.
```
Comment (sabineschaller) [inline docs/src/content/docs/guides/create-interactive-grant.mdx:112]: I think we should also add a response example here

File: docs/src/content/docs/guides/create-interactive-grant.mdx (hunk 1 above)
Comment (melissahenderson) [reply docs/src/content/docs/guides/create-interactive-grant.mdx:63]: This note (OP only supports redirect) appears a few times in this doc, so I'll change each appearance to include "at this time".

File: docs/src/content/docs/guides/create-interactive-grant.mdx (hunk 4 above)
Comment (melissahenderson) [reply docs/src/content/docs/guides/create-interactive-grant.mdx:112]: Will do

File: docs/src/content/docs/guides/create-interactive-grant.mdx [hunk 5]
```diff
@@ -31,80 +32,143 @@ Although grants for creating incoming payments and quotes are non-interactive by
...
+/>
 
-### 3. Interact with the end user
+#### Start interaction with the end user
 
-#### Start interaction
+Once your client receives the authorization server’s response, it must send the end-user to the `interact.redirect` URI contained in the response. This starts the interaction flow.
 
-Once your client receives the [authorization server’s response](/apis/auth-server/operations/post-request/#responses), it must send the end-user to the `interact redirect` URI contained in the response. This initiates the interaction flow.
+<CodeBlock title="Example response">
```
Comment (melissahenderson) [inline docs/src/content/docs/guides/create-interactive-grant.mdx:86]: In the preview, there's white space between this code block's title and body. I don't know why it's happening, so I'll need to investigate more.

File: docs/src/styles/openpayments.css [hunk 6]
```diff
@@ -13,6 +13,11 @@
   max-width: 48rem;
 }
 
+/* Odd one-off margin issue with expressive code (will hoist if seen again) */
```
Comment (huijing) [inline docs/src/styles/openpayments.css:16]: I'm not sure why there is a random insertion of script and link tag in between the container and the figure element containing the code but it is messing up the specificity of the margin-top for that block of code only. I'm unable to see it in other pages so I'm putting in this stop-gap in the OpenPayments-specific styles. If it ever shows up again in another docs site, I will elevate this fix to the shared theme library.
@melissahenderson if this is good enough for merging let me know, I also just pushed into your branch directly. please forgive the intrusion.

**Pull request merged**